common/network_constants.cc

```diff
@@ -8,6 +8,10 @@
 const char kBraveProxyPattern[] = "https://*.brave.com/*";
 const char kBraveSoftwareProxyPattern[] = "https://*.bravesoftware.com/*";
 
+const char kBraveUsageServer[] = "laptop-updates.brave.com";
```

Comment:
this should be defined in config.js/declare_args

Reply:
just like https://github.com/brave/brave-core/pull/6660/files#diff-84f8700538180eec90bb2f4c9dff4da4R452